packages/components/src/tooltip/test/index.js

```diff
@@ -1,371 +1,357 @@
-			act( () => jest.advanceTimersByTime( MOUSE_LEAVE_DELAY ) );
+			await new Promise( ( resolve ) =>
+				setTimeout( resolve, MOUSE_LEAVE_DELAY )
+			);
 
-			// Hover the other button, meaning that the mouse will leave the tooltip anchor
-			await user.hover( externalButton );
+			expect(
+				screen.queryByText( 'tooltip text' )
+			).not.toBeInTheDocument();
+
+			// Hover the other button, meaning that the mouse will leave the tooltip trigger
+			await user.hover(
+				screen.getByRole( 'button', {
+					name: 'Hover me instead!',
+				} )
+			);
 
 			// Tooltip still hasn't appeared yet
-			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
+			expect(
+				screen.queryByText( 'tooltip text' )
+			).not.toBeInTheDocument();
 			expect( onMouseEnterMock ).toHaveBeenCalledTimes( 1 );
 			expect( onMouseLeaveMock ).toHaveBeenCalledTimes( 1 );
 
 			// Advance time again, so that we reach the full TOOLTIP_DELAY time
-			act( () => jest.advanceTimersByTime( TOOLTIP_DELAY ) );
+			await new Promise( ( resolve ) =>
+				setTimeout( resolve, TOOLTIP_DELAY )
+			);
 
-			// Tooltip won't show, since the mouse has left the anchor
-			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
+			// Tooltip won't show, since the mouse has left the tooltip trigger
+			expect(
+				screen.queryByText( 'tooltip text' )
+			).not.toBeInTheDocument();
 		} );
 
 		it( 'should render the shortcut display text when a string is passed as the shortcut', async () => {
-			const user = userEvent.setup( {
-				advanceTimers: jest.advanceTimersByTime,
-			} );
+			const user = userEvent.setup();
 
 			render(
-				<Tooltip text="Help text" shortcut="shortcut text">
-					<button>Hover Me!</button>
+				<Tooltip { ...props } shortcut="shortcut text">
+					<Button>Hover Me!</Button>
 				</Tooltip>
 			);
 
-			const button = screen.getByRole( 'button', { name: 'Hover Me!' } );
-			await user.hover( button );
+			await user.hover(
+				screen.getByRole( 'button', { name: 'Hover Me!' } )
+			);
 
-			const tooltip = await screen.findByText( 'shortcut text' );
-			expect( tooltip ).toBeVisible();
+			await waitFor( () =>
+				expect( screen.getByText( 'shortcut text' ) ).toBeVisible()
+			);
 
 			// Wait for the tooltip element to be positioned (aligned with the button)
 			await waitFor( () =>
 				expect(
-					getWrappingPopoverElement( tooltip )
+					getWrappingPopoverElement(
+						screen.getByText( 'shortcut text' )
+					)
 				).toBePositionedPopover()
 			);
 		} );
 
 		it( 'should render the shortcut display text and aria-label when an object is passed as the shortcut with the correct properties', async () => {
-			const user = userEvent.setup( {
-				advanceTimers: jest.advanceTimersByTime,
-			} );
+			const user = userEvent.setup();
 
 			render(
 				<Tooltip
-					text="Help text"
+					{ ...props }
 					shortcut={ {
-						display: 'shortcut text',
-						ariaLabel: 'shortcut label',
+						display: '⇧⌘,',
```

Comment:
I changed the values of the shortcut object to test the shortcut's purpose, which is to render keyboard shortcuts and corresponding aria-labels for screenreaders.